.github/workflows/warn-on-editing-requirements.yml

```diff
@@ -9,7 +13,9 @@ permissions:
 
 jobs:
   check:
-    if: github.event.review.state == 'APPROVED'
+    if: |
+      github.event_name == 'pull_request_target'
+      || github.event.review.state == 'APPROVED'
```

Comment:
due to the `||`, this condition should only be evaluated if the event name is not `pull_request_target`, which means it should be `pull_request_review`